frontend/amundsen_application/static/js/pages/HomePage/index.tsx

```diff
@@ -27,13 +27,57 @@ export interface DispatchFromProps {
   searchReset: () => UpdateSearchStateReset;
 }
 
+export type Widget = {
+  // TODO: What file does the Widget type belong in?
+  name: string;
+  options: {
+    path: '/';
+  };
+};
+
+export type HomePageLayout = Widget[];
+
 export type HomePageProps = DispatchFromProps & RouteComponentProps<any>;
 
 export class HomePage extends React.Component<HomePageProps> {
   componentDidMount() {
     this.props.searchReset();
   }
 
+  getHomePageWidgets(layout: HomePageLayout): React.ReactNode[] {
+    /* TODO: Make 100% sure it returns the pre-existing layout absent a custom config,
+    to not break any OSS users' Amundsen implementations */
+
+    return [
+      <div className="home-element-container">
+        <SearchBar />
```

Comment:
These need to be updated to the Widgets, right?

Reply:
Yes, I've now updated the component names